test/airport.spec.js

```diff
@@ -0,0 +1,223 @@
+const { assertEquals } = require('../test/testing-frameworks');
+const Airport = require('../src/airport');
+const Plane = require('../src/plane');
+
+function stubIsStormy() {
+    return false;
+}
+
+// Test 1
+console.log('');
+console.log('Test 1:');
+console.log('Land a plane at the airport using landPlane and expect airportPlanes array to increase in length to 1');
+console.log('------------------------------------------------------------------------------------------------------');
+
+// Arrange
+const airport = new Airport();
+const plane1 = new Plane('Plane A');
+
+// Act
+airport.landPlane(plane1);
+const actual = airport.airportPlanes.length;
+const expected = 1;
+
+// Assert
+const result = assertEquals(actual, expected);
+
+// Report
+console.log(`1 plane landed at the airport: ${result}`);
+console.log(`Result: ${result ? 'PASS' : 'FAIL'}`);
+
+// Clean Up
+airport.airportPlanes = [];
+console.log('');
```

Comment:
Even better if

Instead of cleaning up every test manually, I could call the setup function at the beginning of each test and the teardown function at the end of each test.